sdc/tests/test_series.py

```diff
@@ -1182,6 +1182,22 @@ def test_impl(A):
         S = pd.Series(['1', '4', '6', '33', '7'], ['1', '3', '5', '4', '7'], name='A')
         pd.testing.assert_series_equal(hpat_func(S), test_impl(S))
 
+    @unittest.skip('LLVM ERROR: out of memory')
+    def test_series_getitem_idx_int_negative_slice(self):
+        def test_impl(S, start, end):
+            return S[start:end]
+
+        jit_impl = self.jit(test_impl)
+
+        starts = [0, -2]
+        ends = [-2, 0]
+        indices = [[2, 3, 5, 6, 7], ['2', '3', '5', '6', '7'], ['2', '3', '5', '6', '7']]
+        for start, end, index in zip(starts, ends, indices):
```

Comment:
This would iterate over zipped sequences but till the end of the shortest one, so the last string index in indices  won't be tested at all.